dist/Leaflet.BlurredLocation.js

```diff
@@ -13379,6 +13378,27 @@ BlurredLocation = function BlurredLocation(options) {
     }
   }
 
+  function gridWidthInPixels(degrees) {
+    var p1 = L.latLng(1,1);
```

Comment:
Here, we can't just use 1,1 because that's a specific place on the map. We have to start with the current center -- so getLat() and getLon(), then add to those. It'll change based on where in the world you are, remember?